Makefile

```diff
@@ -61,14 +62,14 @@ bin/tigera-operator-$(GIT_VERSION).tgz: bin/helm $(shell find ./charts/tigera-op
 
 # Build all Calico images for the current architecture.
 image:
```

Comment:
This is actually just a dev target, and isn't used as part of the release process.

Reply:
Reverted this change.